packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -90,10 +95,37 @@ export default class QuanticResultLink extends LightningElement {
     );
   };
 
+  handleOpenTabSalesforceLink(event) {
+    event.stopPropagation();
+    const targetPageRef = {
+      type: 'standard__recordPage',
+      attributes: {
+        recordId: this.getRecordIdAttribute(),
+        actionName: 'view',
+      },
+    };
+    this[NavigationMixin.Navigate](targetPageRef);
+  }
+
+  getRecordIdAttribute() {
```

Comment:
just as an FYI, LWC has this special way to implement getters: https://salesforcediaries.com/2020/10/12/understand-getter-in-lightning-web-component/